docs/errors.md

```diff
@@ -60,6 +60,7 @@ Each error code is defined by a constant in the `twirp` package:
 | Internal           | internal            | 500 Internal Server Error
 | Unavailable        | unavailable         | 503 Service Unavailable
 | DataLoss           | dataloss            | 500 Internal Server Error
+| TooManyRequests    | too_many_requests   | 429 Too Many Requests
```

Comment:
Note that the errors at the bottom are all 5xx errors. Would you mind moving the new definition to someplace above? I think next to ResourceExhausted is best, that will help gRPC users to see the specific code that Twirp uses next to the one they use.